2. flow_control/Q15.c

```diff
@@ -1 +1,26 @@
 // Write a program to read an array of n integers and print the array in reverse order using a for loop.
+#include<stdio.h>
+int main(){
+    int n,array[10],i;
+    printf("Enter the number of elements in the array = ");
+    scanf("%d", &n);
+    
+    for ( i = 0; i <= n-1; i++) //n-1 because array start from 0 and goes till n-1
```

Comment:
you can use `i<n` also 
test the input for n>10 and check the result.

Reply:
for n>10, it will show error since array[10] can only have 10 elements.